playground/LoadTest/LoadTest.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net461</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>1701;1702;0003;CS0108;CS0109;CS0169;CS0067;IDE0060;IDE0022;IDE0052;IDE0044;IDE0051;IDE0059</NoWarn>
```

Comment:
I ignored lot of warnings because they are not relevant for this project (unused properties / fields, etc )